siac/walletcmd.go

```diff
@@ -224,8 +227,11 @@ func walletloadseedcmd() {
 }
```

Comment:
Not touched by this PR, but `fmt.Println("Added Key")` could me reworded to be more consistent with the other success indications. It's especially confusing because the word key is used instead of seed.

Even the existing success messages could be improved. Maybe our resident grammar master could comment, but @lukechampine "Wallet loading successful." doesn't seem like a proper sentence.